### ISSUE TITLE
Installation of RcisTarget.hg19.motifDatabases.20k_0.1.1.tar.gz

### ISSUE THREAD

**Issue by rashtoncta** [aertslab/RcisTarget#4]
Hi,

I'm trying to install the human RcisTarget database by running 
`install.packages("http://scenic.aertslab.org/downloads/databases/RcisTarget.hg19.motifDatabases.20k_0.1.1.tar.gz",repos = NULL,type="source")`

This is failing with the following error
```
* installing *source* package ‘RcisTarget.hg19.motifDatabases.20k’ ...
** data
** help
Error : /tmp/RtmpzTK61I/R.INSTALL2f9b640c131c/RcisTarget.hg19.motifDatabases.20k/man/RcisTarget.hg19.motifDatabases.Rd: Sections \title, and \name must exist and be unique in Rd files
ERROR: installing Rd objects failed for package ‘RcisTarget.hg19.motifDatabases.20k’
* removing ‘/usr/local/lib/R/site-library/RcisTarget.hg19.motifDatabases.20k’
Warning message:
In install.packages("http://scenic.aertslab.org/downloads/databases/RcisTarget.hg19.motifDatabases.20k_0.1.1.tar.gz",  :
  installation of package ‘/tmp/Rtmp3bUWLP/downloaded_packages/RcisTarget.hg19.motifDatabases.20k_0.1.1.tar.gz’ had non-zero exit status
```

I've successfully installed the mouse database using a similar call
`install.packages("http://scenic.aertslab.org/downloads/databases/RcisTarget.mm9.motifDatabases.20k_0.1.1.tar.gz",repos = NULL,type="source")`

Any ideas how I can resolve this? 

Thanks in advance.

**Comment by s-aibar** [comment]
Hello,

I just ran that command and it has worked, so it might have something to do with the R version or settings/setup... Which R version are you using? 

There is only one .Rd file in the package, and it does have /name and /title fields... 
Can you try  "uninstalling" the package (e.g. uninstall.packages("RcisTarget.mm9.motifDatabases.20k") and/or uninstall.packages("RcisTarget.mm9.motifDatabases")) to see whether the next attempt works...?

If not, then let me know your R version, or any other thing you think it might be affecting...

**Comment by rashtoncta** [comment]
Hi,

Thanks for help with this.

I've managed to add the package to R by downloading the tar ball manually and copying this to the server I was trying to install on. I then installed it from the directory using `install.packages('RcisTarget.hg19.motifDatabases.20k' , repos=NULL, type="source")`.

Before this I tried removing any existing RcisTarget databases as you suggested, I also cleared up any R tmp directories but I still ran into the same problems of R complaining about \title and \name sections in Rd files. I think it is likely this was some problem on the particular server. This was with R 3.3.3 (2017-03-06) -- "Another Canoe".

This can be closed as I think it was a problem with my machine. Thanks for your help.
